Project.toml

```diff
@@ -0,0 +1,20 @@
+name = "CondaBinDeps"
+uuid = "a9693cdc-2bc8-5703-a9cd-1da358117377"
+version = "0.2.0"
+
+[deps]
+BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Do we still need `Compat` since we are now requiring Julia 1.x?

Reply:
You are right, Compat can indeed be dropped. I adapted the PR.